src/parser/jobs/rdm/modules/Procs.js

```diff
@@ -36,29 +61,55 @@ export default class Procs extends Module {
 	_playerWasInDowntime = false
 	_lastTargetID = 0
 
+	_buffWindows = {
+		[STATUSES.VERFIRE_READY.id]: {
+			current: null,
+			history: [],
+		},
+		[STATUSES.VERSTONE_READY.id]: {
+			current: null,
+			history: [],
+		},
+	}
+
 	constructor(...args) {
 		super(...args)
 
 		this.addHook('cast', {by: 'player'}, this._onCast)
-		this.addHook('applybuff', {
-			to: 'player',
-			abilityId: PROCS,
-		}, this._onGain)
-		this.addHook('removebuff', {
-			to: 'player',
-			abilityId: PROCS,
-		}, this._onRemove)
-		this.addHook('refreshbuff', {
-			to: 'player',
-			abilityId: PROCS,
-		}, this._onRefresh)
+		this.addHook('applybuff', {by: 'player', abilityId: PROCS}, this._onGain)
+		this.addHook('removebuff', {by: 'player', abilityId: PROCS}, this._onRemove)
+		this.addHook('refreshbuff', {by: 'player', abilityId: PROCS}, this._onRefresh)
 		this.addHook('complete', this._onComplete)
+		//this.addHook('death', { to: 'player' }, this._onDeath)
```

Comment:
Is this something planned, or left over from experimenting that should be removed?

Reply:
It was from initial experiments, but with how I finished out onComplete I have what I need for onDeath for the timeline at least.  So I've implemented it for now; it'll be in this upcoming commit.